Packs/BreachRx/Integrations/BreachRx/BreachRx.py

```diff
@@ -0,0 +1,301 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from collections.abc import Callable
+
+import requests
+import traceback
+
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+from requests.auth import HTTPBasicAuth
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+create_incident_mutation = gql("""
+mutation CreateIncident(
+  $severity: String!,
+  $name: String!,
+  $type: String!,
+  $description: String
+) {
+  createIncident(
+    type: $type,
+    severity: $severity,
+    name: $name,
+    description: $description
+  ) {
+    id
+    name
+    severity {
+      name
+    }
+    types {
+      type {
+        name
+      }
+    }
+    description
+    identifier
+  }
+}""")
+
+get_incident_severities = gql("""{
+  incidentSeverities {
+    name
+    ordering
+  }
+}""")
+
+get_incident_types = gql("""{
+  types {
+    name
+  }
+}""")
+
+get_actions_for_incident = gql("""
+query GetActionsForIncident($incidentId: Int!) {
+  actions(where: {
+    incidentId: {
+      equals: $incidentId
+    }
+  }) {
+    id
+    name
+    description
+    phase {
+      name
+      id
+    }
+    user {
+      fullName
+      email
+    }
+  }
+}""")
+
+get_incident_by_name = gql("""
+query GetIncidentByName($name: String, $identifier: String) {
+  incidents(first: 1, where: {
+    name: {
+      contains: $name,
+      mode: insensitive
+    }
+    identifier: {
+      contains: $identifier,
+      mode: insensitive
+    }
+  }) {
+    id
+    name
+    severity {
+      name
+    }
+    types {
+      type {
+        name
+      }
+    }
+    description
+    identifier
+  }
+}""")
+
+
+class BreachRxClient:
```

Comment:
I can, though I did intentionally avoid this because `BaseClient` seems specific to the `requests` library, and this integration uses `requests` in a relatively unique way (by using the `gql` `Client` class and passing in a gql specific `RequestsHTTPTransport` instance, instead of the `requests.Session` class).

I could probably rewrite this to allow use of the member functions in `BaseClient` but it doesn't seem like the BreachRx integration needs them right now.  If this is important I'll get to work, otherwise I'd prefer to leave it as-is, if that's okay!

Reply:
Just as an update, I did end up having this class inherit `BaseClient` and everything seems to be working still, so if that's all you're looking for that seems fine.

Reply:
My suggestion is only a recommendation if you have a good reason to revert the class you can do it.
It's up to you.